src/Controls.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component, Fragment } from 'react';
+import './controls.css';
+
+class Controls extends Component {
+  render() {
+    return (
+      <Fragment>
+        <div className="controls">
+          <div className="controls-title">Instructions to Play</div>
+          <ol>
+
+            <li>1. Press <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="36" height="36">
+            <path fill="none" d="M0 0h24v24H0z" />
+            <path d="M8 12l6-6v12z" fill="rgba(255,255,255,1)" />
+            </svg> to Move Left.</li>
+
+            <li>2. Press <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="36" height="36">
```

Comment:
Use prettier to format the code. Configuration- trailing comma es5, single quotes always.

So this should be

```html
<li>
  Press
  <svg>